src/ExEss/Cms/FLW_Flows/Response/Model.php

```diff
@@ -125,7 +127,7 @@ public function jsonSerialize(): array
      */
     public function toArray(): array
```

Comment:
what is this used for? Might be a next step to get rid of this

Reply:
there were still quite a bit usages where we needed the array representation instead of the object, will have a look if we still do